OrbitCore/SamplingProfiler.h

```diff
@@ -166,6 +170,8 @@ class SamplingProfiler {
       m_OriginalCallstackToResolvedCallstack;
   std::unordered_map<uint64_t, std::set<CallstackID>> m_FunctionToCallstacks;
   std::unordered_map<uint64_t, uint64_t> m_ExactAddressToFunctionAddress;
+  std::unordered_map<uint64_t, std::set<uint64_t>>
+      m_FunctionAddressToExactAddresses;
```

Comment:
Is `std::set<uint64_t>` needed or can a `std::unordered_set<uint64_t>` be used? I've seen you're iterating over it, but I'm not sure if the ordering is important.

Reply:
Changed it to an `unordered_set`. Thanks!